news/2025/2025-10-30-fall-time-mania-mapping-contest.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Fall Time Mania Mapping Contest 2025
+date: 2025-10-30 16:00:00 +0000
+---
+
+The first iteration of FTMC is on its way with the chilly autumn air! Are you ready to start your holiday mapping?
+
+![](wiki/shared/news/2025-10-30-fall-time-mania-mapping-contest/banner.jpg)
+
+With the temperatures dropping rather quickly, we would like to give you a warm welcome to the **Fall Time Mania Mapping Contest**'s debut! You can read the full details in the forum post linked [here](https://osu.ppy.sh/community/forums/topics/2145873?n=1).
+
+This mapping contest originated from its close seasonal partner, [STMI](https://osu.ppy.sh/community/forums/topics/2050785?n=1), which was an osu!mania 6K contest whose goal was getting more players interested in the keymode. After the contest's success, we now want to welcome all mappers by allowing submissions **from 4 to 10 keys!**
+
+FTMC 2025 will be a one difficulty, one song per submission contest from osu!'s Featured Artist listing. It will follow a specific judging criteria listed below with the inclusion of a gimmick. Judging criteria and rules also expanded on the [contest's forum post found here](https://osu.ppy.sh/community/forums/topics/2145873?n=1). How should you map? Well, the editor is your canvas! You're to map as your heart desires as long as you follow the rules!
+
+## Schedule
+
+Please note that any schedule changes will be made known in the contest's [Discord server](https://discord.gg/p8bwdrjExr) as well as through the forum post. Please keep an eye out on these outlets to receive the most up-to-date news regarding the contest.
+
+| Event | Dates |
+| --: | :-- |
+| Beatmapping (3.5 weeks) | October 30th - November 24th (16:00 UTC) |
+| Judging (2 weeks) | November 24th - December 8th (16:00 UTC) |
+| Results (1 week) | December 9th - December 15th |
+
+Results will be made public through the Discord server as well as through a conclusion news post.
+
+## Prizes
+
+This contest comes packed with prizes so do not fret! Your efforts will be rewarded!
+
+We've prepared some lovely prizes to reward those who impress our judges the most! We also have a category called **Judge's Pick** which will feature the entry that the judges deemed worthy of an extra prize!
```

Comment:
what? the regular prizes are decided by the same judges as well, no? how is this one picked?